### PR TITLE
Use delimiter dropdown

### PR DESCRIPTION
**Pull request by Sjoerd1993**
Adds a dropdown menu for the delimiter choices with more understandable options. When a custom delimiter is chosen, an entry row is shown where the delimiter can be specified manually. Closes issue #627 

image

### CODE REVIEW

File: src/migrate.py [hunk 1]
```diff
@@ -77,6 +77,8 @@ def _migrate_import_params(settings_, import_file):
         for key, value in params.items():
             if key == "separator":
                 settings.set_string(key, f"{value} ")
+            if key == "delimiter":
+                settings.set_string("custom-delimiter", value)
```
Comment (cmkohnen) [inline src/migrate.py:81]: This should check against the Delimiters in misc first

File: src/migrate.py [hunk 2]
```diff
@@ -77,6 +77,9 @@ def _migrate_import_params(settings_, import_file):
         for key, value in params.items():
             if key == "separator":
                 settings.set_string(key, f"{value} ")
+            if key == "delimiter":
+                settings.set_string("custom-delimiter",
+                                    misc.get_delimiter(settings))
```
[the remark this answers is not in the document]
Comment (Sjoerd1993) [reply src/migrate.py:82]: Wait yeah, that's indeed silly. It should just set "custom-delimiter" in the new settings to whatever "delimiter" was in the old project file. 

I'm not at my computer right now, but looking at it I think the old logic where it sets it to `value` should be correct.

**Comment by Sjoerd1993** [comment]
Looked at it, and I see the initial point now. Initially I just set the custom delimiter to the old value. But it's much prettier of course to set the value to one of the preset options if there's a match. Witch this does now, properly.

In testing this with on old Project, I noticed there's a typo in the old Graphs 1.6.1 branch where it says `mix_right` instead of `min_right`. This makes it impossible to open old Projects as it will lead to a key error. So that has been added to the migration table as well.

**Comment by cmkohnen** [comment]
Oops, accidentally pushed directly. Opened up #634 which cleans up the code around the import migration a bit

**Review by cmkohnen — APPROVED** [review]

**Pull request merged**